### DISCUSSION TITLE
adding a markdown

### DISCUSSION THREAD

**Discussion by Hezethegamer** [community/community#84755] [New to GitHub] [answered]
i have created a `README.md` and wanted it to appear to have been written by markdown , how can i do that

**Comment by rezzcode** [comment, marked as answer]
hello...
if you want your readme to appear to have been written in a Markdown language the probably you may want to do the following.

1. create a file called `.gitattributes`
2. add the following lines to it:

```md
# Markdown
*.md linguist-detectable=true
*.md linguist-documentation=false
```

The above will help you achieve what you want, thanks..

**Reply by Hezethegamer** [reply]
let me try this out.

**Reply by rezzcode** [reply]
yeah, sure! no problem.

**Reply by Hezethegamer** [reply]
thanks for your efforts to help, worked

![image](https://github.com/Hezethegamer/alx-higher_level_programming/assets/122324108/9fdb71f6-b80c-496b-8286-09a064052f60)

**Reply by rezzcode** [reply]
no problem, good to hear it worked.

**Reply by ibrahimkhled** [reply]
# Markdown
*.md linguist-detectable=true
*.md linguist-documentation=false

**Reply by mishmanners** [reply]
This is a great example of an awesome answer ❤️

**Reply by sketchprodigal37266** [reply]
# Markdown
*.md linguist-detectable=true
*.md linguist-documentation=false

**Reply by rezzcode** [reply]
> This is a great example of an awesome answer ❤️

Thanks for the feedback. :smile:

**Reply by Hezethegamer** [reply]
@mishmanners indeed it is, his idea worked .

**Reply by Hakosako** [reply]
1. Karasuh485@gmail.com #10138865### 
![Uploading Screenshot_20240113_232247_MetaMask.jpg…]()

**Comment by akashdip2001** [comment]
To make your README.md appear as if it's written in Markdown on GitHub, you need to use Markdown syntax. Here are some basic Markdown formatting examples you can include in your README:

1. **Headers:**
   ```markdown
   # Main Header
   ## Subheader
   ### Sub-subheader
   ```

2. **Emphasis (Bold and Italics):**
   ```markdown
   **Bold Text**
   *Italic Text*
   ```

3. **Lists:**
   ```markdown
   - Item 1
   - Item 2
     - Subitem A
     - Subitem B
   1. Numbered Item 1
   2. Numbered Item 2
   ```

4. **Links:**
   ```markdown
   [Link Text](https://www.example.com)
   ```

5. **Images:**
   ```markdown
   ![Alt Text](image-url.png)
   ```

6. **Code Blocks:**
   ```markdown
   ```
   Your code here
   ```
   ```

7. **Quotes:**
   ```markdown
   > This is a quote.
   ```

8. **Tables:**
   ```markdown
   | Header 1 | Header 2 |
   | -------- | -------- |
   | Content 1| Content 2|
   ```

9. **Horizontal Rules:**
   ```markdown
   ---
   ```

10. **Checkboxes:**
    ```markdown
    - [x] Task 1
    - [ ] Task 2
    ```

These are just basic examples; you can combine and extend them as needed. When you commit and push your README.md to your GitHub repository, GitHub will render it as Markdown, and you'll see the formatted content.

Remember that the rendering may vary slightly between platforms, but these Markdown basics should work well on GitHub.

## Profile
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
"everything is physics bro"  
E=mc²

|[Subscribe on YouTube](https://www.youtube.com/channel/UCxvmp634YDc41xCWOdvWqoQ?sub_confirmation=1) |
|----  |

**Reply by rezzcode** [reply]
this is a good one

**Comment by New-templates1990** [comment]
> hello... if you want your readme to appear to have been written in a Markdown language the probably you may want to do the following.
> 
> 1. create a file called `.gitattributes`
> 2. add the following lines to it:
> 
> ```
> # Markdown
> *.md linguist-detectable=true
> *.md linguist-documentation=false
> ```
> 
> The above will help you achieve what you want, thanks..

**Comment by Stellarhold170NT** [comment]
I recommend reading the book The Mark Down Guide by Matt Cone
​
